tracing-opentelemetry/README.md

```diff
@@ -59,22 +59,25 @@ The crate provides the following types:
 ### Basic Usage
 
 ```rust
-use opentelemetry::{api::Provider, sdk};
+use opentelemetry::exporter::trace::stdout;
 use tracing::{error, span};
 use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::Registry;
 
 fn main() {
-    // Create a new tracer
-    let tracer = sdk::Provider::default().get_tracer("component_name");
+    // Install a new OpenTelemetry trace pipeline
+    let (tracer, _uninstall) = stdout::new_pipeline().install();
```

Comment:
Maybe add a short explanation here what this does with stdout?

Reply:
Are you interested in the results? e.g. closed spans will now be sent to stdout, can be replaced by otel compatible exporters, etc, or what all installing a pipeline configures for you? e.g. crates a tracer provider with the recommended defaults and sets it as the global provider?

Reply:
I was a little confused that this is using `stdout::new_pipeline()` (which I also hadn't seen before) whereas `examples/opentelemetry.rs` uses Jaeger. Why use something different for these two cases?

Reply:
The thinking here was that the main README was likely the first place someone would interact with this crate so the stdout pipeline shows the minimal requirements to start exporting data. Once someone gets to the code in `examples/opentelemetry.rs` they likely have a little more context so it is a more real world example that requires the additional steps of setting up `jaeger` locally for it to produce any visible results.

Reply:
Fair enough! Would be nice to get this out.